app/src/main/java/org/sopt/at/utils/Keyboard.kt

```diff
@@ -0,0 +1,10 @@
+package org.sopt.at.utils
+
+import android.app.Activity
+import android.content.Context
+import android.view.inputmethod.InputMethodManager
+
+fun hideKeyboard(activity: Activity){
```

Comment:
입력이 끝났을때 키보드를 내리려고 만든거지요?
키보드는 컴포저블에 포커스가 생기면 올라오고, 포커스가 사라지면 내려갑니다.
그래서 굳이 함수로 안만들고 최상위 컴포저블에서 포커스를 해제시켜주면 됩니다.
```suggestion
val focusManager = LocalFocusManager.current
focusManager.clearFocus()
```